Reading_files/vascular.ipynb

```diff
@@ -1,5 +1,16 @@
 {
```

Comment:
<p>Why did you use <code>join='outer'</code>?  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>